content/en/docs/concepts/overview/working-with-objects/_index.md

```diff
@@ -1,7 +1,150 @@
 ---
-title: "Working with Kubernetes Objects"
-weight: 40
+title: Objects in Kubernetes
+content_type: concept
+weight: 10
+card: 
+  name: concepts
+  weight: 40
 description: >
   Kubernetes objects are persistent entities in the Kubernetes system. Kubernetes uses these entities to represent the state of your cluster.
   Learn about the Kubernetes object model and how to work with these objects.
 ---
+
+<!-- overview -->
+This page explains how Kubernetes objects are represented in the Kubernetes API, and how you can
+express them in YAML format (a _manifest_).
+
+
+<!-- body -->
+## Understanding Kubernetes objects {#kubernetes-objects}
+
+*Kubernetes objects* are persistent entities in the Kubernetes system. Kubernetes uses these
+entities to represent the state of your cluster. Specifically, they can describe:
+
+* What containerized applications are running (and on which nodes)
+* The resources available to those applications
+* The policies around how those applications behave, such as restart policies, upgrades, and
+  fault-tolerance
+
+A Kubernetes object is a "record of intent"—once you create the object, the Kubernetes system
```

Comment:
```suggestion
A Kubernetes object is a "record of intent" — once you create the object, the Kubernetes system
```

Reply:
Not sure what our style guide says about em dashes. Using em dashes without surrounding space is usual for UK typography.